_layouts/faq.html

```diff
@@ -1,16 +1,5 @@
 ---
```

Comment:
When running with `make serve-quick`, it looks this way:

![Screenshot from 2021-03-18 11-06-27](https://user-images.githubusercontent.com/1842467/111618975-4e8c2500-87e5-11eb-84a8-fa7a14cb9d2b.png)